cypress/e2e/course/createEditStopSyncedCourse.feature

```diff
@@ -141,13 +140,10 @@ Feature: Teacher can create, update and stop a synchronized course
       |  Cypress-Test-Existing-Course  | Cypress-Test-Group             |
 
   Scenario Outline: Stop a course synchronization in the class overview
-    When I click on administration in menu
```

Comment:
Why do you change tests for the class overview, when you just worked in the administration course overview page?

Reply:
this is actually the course sync 😅 I had to make changes there, because we changed the stop sync modal (removal of the warning) and this test would fail otherwise. While i was at it i fixed a few things :)